docs/user/adding-pdf-annotations.md

```diff
@@ -170,6 +170,42 @@ with open("annotated-pdf.pdf", "wb") as fp:
     writer.write(fp)
 ```
 
+## Popup
+
+Manage the Popup windows for markups. looks like this:
+
+![](annotation-popup.png)
+
+you can use the {py:class}`AnnotationBuilder <pypdf.generic.AnnotationBuilder>`:
+
+you have to use the returned result from add_annotation() to fill-up the
+
+```python
+# Arrange
+writer = pypdf.PdfWriter()
+writer.append(os.path.join(RESOURCE_ROOT, "crazyones.pdf"), [0])
+
+# Act
+text_annotation = writer.add_annotation(
+    0,
+    AnnotationBuilder.text(
+        text="Hello World\nThis is the second line!",
+        rect=(50, 550, 200, 650),
+        open=True,
+    ),
+)
+
+popup_annotation = AnnotationBuilder.popup(
+    rect=(50, 550, 200, 650),
+    open=True,
+    parent=text_annotation,  # use the output of add_annotation
+)
+
+target = "annotated-pdf-popup.pdf"
+writer.write(target)
+os.remove(target)  # comment this out for manual inspection
```

Comment:
```suggestion
```

Reply:
Can you clarify why?

Reply:
This line makes sense in the test, but why would somebody who uses the code want to directly remove the result?

Reply:
I'm reusing the same approach as in test_workflows.py#L=337

Reply:
Yes, and that is fine. But in contrast to the test code, the user wants to see the file with the annotation. Why would a user delete the PDF after annotating it?